### PR TITLE
kodi: another december update

### PR DESCRIPTION
**Pull request by MilhouseVH**
Includes fix for pvr.zattoo, dropped from #4070

### CODE REVIEW

File: packages/mediacenter/kodi-binary-addons/visualization.projectm/package.mk [hunk 1]
```diff
@@ -3,8 +3,8 @@
 # Copyright (C) 2018-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="visualization.projectm"
-PKG_VERSION="2.3.2-Leia"
-PKG_SHA256="e30163b4eb12518deb1b56c488b0a81f7eeb3467d291dad6e28234843fc9dc40"
+PKG_VERSION="2.3.4-Leia"
```
Comment (CvH) [inline packages/mediacenter/kodi-binary-addons/visualization.projectm/package.mk]: there is a matrix branch (+ release)

File: packages/mediacenter/kodi-binary-addons/screensaver.stars/package.mk [hunk 2]
```diff
@@ -3,9 +3,9 @@
 # Copyright (C) 2018-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="screensaver.stars"
-PKG_VERSION="2.1.2-Leia"
-PKG_SHA256="8873de9be41365af9dc639e3624d887f7b18d8e1f43e9a69af0c3154f9b59020"
-PKG_REV="2"
+PKG_VERSION="2.1.3-Leia"
```
Comment (CvH) [inline packages/mediacenter/kodi-binary-addons/screensaver.stars/package.mk]: there is a matrix branch (+ release)

File: packages/mediacenter/kodi-binary-addons/pvr.freebox/package.mk [hunk 3]
```diff
@@ -2,8 +2,8 @@
 # Copyright (C) 2019-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="pvr.freebox"
-PKG_VERSION="2.2.0-Matrix"
-PKG_SHA256="4195031c9bc63077cf03ce445679e8ab9a6b508c6b28ec842c296ce925966631"
+PKG_VERSION="2.0.1-Leia"
```
Comment (CvH) [inline packages/mediacenter/kodi-binary-addons/pvr.freebox/package.mk]: wrong branch ?

File: packages/mediacenter/kodi/package.mk [hunk 4]
```diff
@@ -13,8 +13,8 @@ PKG_PATCH_DIRS="$KODI_VENDOR"
 
 case $KODI_VENDOR in
   raspberrypi)
-    PKG_VERSION="5f134adfcee6fd031c7b8f905bce892dc6d88f7a" # kodi19-pre-Python3
-    PKG_SHA256="9acec0b04f802b8bcd24922a1664ee47198f384b6ba8f0704718dde176a81858"
+    PKG_VERSION="b4e54bf036c7472e6842d5c960a6fdae36ba19f2" # kodi19-pre-Python3
```
Comment (CvH) [inline packages/mediacenter/kodi/package.mk]: not sure we want an 3 year old version ? :)

Comment (MilhouseVH) [reply packages/mediacenter/kodi/package.mk]: Sorry - should be fixed now, thanks for noticing! :)

File: packages/mediacenter/kodi-binary-addons/pvr.freebox/package.mk (hunk 3 above)
Comment (MilhouseVH) [reply packages/mediacenter/kodi-binary-addons/pvr.freebox/package.mk]: Same tag used for both Leia & Matrix releases: https://github.com/aassif/pvr.freebox/releases

File: packages/mediacenter/kodi-binary-addons/screensaver.stars/package.mk (hunk 2 above)
Comment (MilhouseVH) [reply packages/mediacenter/kodi-binary-addons/screensaver.stars/package.mk]: Kodi is still using the Leia branch for this repo: https://github.com/xbmc/repo-binary-addons/blob/Matrix/screensaver.stars/screensaver.stars.txt

File: packages/mediacenter/kodi-binary-addons/visualization.projectm/package.mk (hunk 1 above)
Comment (MilhouseVH) [reply packages/mediacenter/kodi-binary-addons/visualization.projectm/package.mk]: Kodi is still using Leia: https://github.com/xbmc/repo-binary-addons/blob/Matrix/visualization.projectm/visualization.projectm.txt

**Comment by 150balbes** [comment]
No one has faced the problem of installing Addons (no access to all network repositories) with this version of KODI ?

**Pull request merged**